### DISCUSSION TITLE
Merch for Open-Source MuJoCo Community

### DISCUSSION THREAD

**Discussion by peterdavidfagan** [google-deepmind/mujoco#1644] [General]
Not very relevant to actual development work but it would be super cool if MuJoCo had official merch for open-source developers (e.g. hoodies and laptop stickers).

**Comment by yuvaltassa** [comment]
Or perhaps the mug from [model/mug/mug.xml](https://github.com/google-deepmind/mujoco/blob/main/model/mug/mug.xml) ? 🙂 

This is a nice idea actually. I'm sure there is someone I can ask about this... will update here with any developments.

![mug](https://mujoco.readthedocs.io/en/latest/_images/mug.png)

**Reply by peterdavidfagan** [reply]
> Or perhaps the mug from [model/mug/mug.xml](https://github.com/google-deepmind/mujoco/blob/main/model/mug/mug.xml) ? 🙂

This mug but in the og simulator of real life would be very cool. Hahaha would be neat having model files for each piece of merch so it can be simulated in MuJoCo too. 

> This is a nice idea actually. I'm sure there is someone I can ask about this... will update here with any developments.

Cool, yeah it crossed my mind recently that MuJoCo merch would be very cool, thought to post it in case others felt the same.
